packages/stark-ui/src/modules/breadcrumb/components/breadcrumb.component.scss

```diff
@@ -0,0 +1,52 @@
+/* ============================================================================== */
+/*  S t a r k   B r e a d c r u m b                                               */
+/* ============================================================================== */
+/* stark-ui: src/modules/breadcrumb/components/breadcrumb/breadcrumb.component.scss */
+.stark-breadcrumb {
+  display: none;
+}
+
+.stark-breadcrumb {
+  display: block;
+  a {
+    cursor: pointer;
+  }
+}
+
+@media #{$tablet-query} {
+  .stark-breadcrumb {
+    display: block;
+    padding-left: 8px;
+    font-size: 15px;
+    line-height: 24px;
+    margin-bottom: 0;
+    color: #fff;
+    a {
+      display: inline-block;
+      padding: 0 6px;
+      color: var(--tertiary-dark-text-color);
```

Comment:
This variable does not exist anymore (was part of our postcss implementation), please change it for the new one.